sintia/sintia.py

```diff
@@ -261,6 +261,27 @@ def is_rate_limited(self, action: str, user_id: int) -> bool:
             return await message.channel.send(f'No results found for `{argument}`')
 
         return await message.channel.send(search_result["link"])
+    
+    @command_handler('gif')
+    async def google_image_search(self, message: discord.Message , argument: str) -> None:
```

Comment:
you should change the method name to `google_gif_search` to avoid collision with the previous method!